docs/sphinx/user/inputs_Actuator.rst

```diff
@@ -179,6 +164,36 @@ Example for ``FixedWingLine``::
    
    This is how often to write actuator output. The default is ``10``.
 
+.. input_param:: Actuator.FixedWingLine.motion_type
+
+   **type:** String, optional
+
+   Slightly contrary to its name, the FixedWingLine actuator allows for motion, 
```

Comment:
Fixed wing is the correct terminology even if the wing is moving.
I think it is best to remove "Slightly contrary to its name"